koku/masu/external/kafka_msg_handler.py

```diff
@@ -237,6 +238,22 @@ def construct_parquet_reports(request_id, context, report_meta, payload_destinat
     return daily_parquet_files
 
 
+def remove_invalid_csv(dir_path):
+    for (root, _, filenames) in os.walk(dir_path):
+        csv_reports = [fl for fl in filenames if fl.endswith(".csv")]
+        for csv_report in csv_reports:
+            file_path = f"{root}/{csv_report}"
+            try:
+                pd.read_csv(file_path)
```

Comment:
One other option is to call this here instead:
https://github.com/project-koku/koku/blob/262bd28754c6ea57f07c8229924dcc7c01db4d56/koku/masu/util/ocp/common.py#L285

That should not only verify that it's a CSV file but we can also verify that the file is a valid OCP report